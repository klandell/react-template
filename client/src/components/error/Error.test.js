import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Error from './Error';

test('Error component should render a 404 error.', () => {
    const component = shallow(<Error />);
    const tree = toJson(component);
    expect(tree).toMatchSnapshot();
});
