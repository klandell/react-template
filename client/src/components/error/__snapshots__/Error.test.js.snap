// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error component should render a 404 error. 1`] = `
<div
  className="error"
>
  <h1>
    404
  </h1>
</div>
`;
